packages/blockchain-link/src/workers/solana/index.ts

```diff
@@ -10,6 +10,16 @@ export type SolanaAPI = Connection;
 type Context = ContextType<SolanaAPI>;
 type Request<T> = T & Context;
 
+const pushTransaction = async (request: Request<MessageTypes.PushTransaction>) => {
+    const txHash = request.payload.startsWith('0x') ? request.payload.slice(2) : request.payload;
```

Comment:
This is called `txHash`, but is this really just the has of the transaction? I think is is the raw transaction, so maybe you meant `txHex` ?